assignments/prototypes.js

```diff
@@ -15,6 +15,15 @@
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
 
+function GameObject (createdAt, dimensions) {
+  this.createdAt = createdAt,
+  this.dimensions = dimensions,
+  this.destroy = function () {
+    return `${this.name} was removed from the game`;
+  };
```

Comment:
This function is supposed to be added to the prototype. As it is, this function will be recreated every time you create a new instance of this object. By adding it to the prototype, only a reference is created each time which can preserve a lot of memory:
```
GameObject.prototype.destroy = function ()
 {return `${this.name} was removed from the game.`;};
```